UNRELEASED.md

```diff
@@ -0,0 +1,5 @@
+**Added:**
+ - `bpk-component-floating-notification` - `1.0.0`
+
+- bpk-component-price
+  - Price is a new component to ensure consistent display of pricing.
```

Comment:
is this part of this PR?

Reply:
No, but it was part of `main`

Reply:
@olliecurtis Should I only include my changes?

Reply:
If you rebase this it should be gone now as we have released this change since 😄